eng/pipelines/templates/stages/apiview-python-ci.yml

```diff
@@ -0,0 +1,121 @@
+trigger:
+  branches:
+    include:
+      - main
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - eng/pipelines/templates/stages/apiview-python-ci.yml
+      - packages/python-packages/api-stub-generator
```

Comment:
I would still love for us to get these tools somewhere under the tools directory.